build-support/bin/deploy_to_s3.py

```diff
@@ -22,10 +22,10 @@ def main() -> None:
         ),
     )
     options = parser.parse_args()
-    perform_deploy(aws_cli_symlink_path=options.aws_cli_symlink_path, scope=options.scope)
+    perform_deploy(scope=options.scope)
 
 
-def perform_deploy(*, aws_cli_symlink_path: str | None = None, scope: str | None = None):
+def perform_deploy(*, aws_cli_symlink_path: str | None = None, scope: str | None = None) -> None:
```

Comment:
The `aws_cli_symlink_path` param _does_ get used, but the option and use are over here:
https://github.com/pantsbuild/pants/blob/64a480d1f0f5baba678d7f39d69af72d8e096a6c/build-support/bin/backfill_s3_release_tag_mappings.py#L11-L35